lib/StreamParser.js

```diff
@@ -20,26 +30,28 @@ const StreamParser = function (options) {
 
   this.currentState = options.startWithState;
   this.transitions = options.transitions;
+
+  this.pipe(new VoidStream());
 };
 
 util.inherits(StreamParser, Transform);
 
 /* eslint-disable no-underscore-dangle */
```

Comment:
Die neueren ESLint-Versionen verstehen auch:

```suggestion
// eslint-disable-next-line no-underscore-dangle
```

Sieht etwas schöner aus, finde ich.